src/Package/Impl/Editors/BaseEditorFactory.cs

```diff
@@ -318,5 +312,30 @@ public class BaseEditorFactory : IVsEditorFactory, IDisposable {
 
         protected virtual void Dispose(bool disposing) { }
         #endregion
+
+        public static void InitKeyBindings(ITextView textView) {
```

Comment:
Looks more like a hepler or extension method. Maybe move it to the TextViewExtensions?